packages/elastic-datemath/BUILD.bazel

```diff
@@ -4,15 +4,15 @@ load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
 PKG_BASE_NAME = "elastic-datemath"
 PKG_REQUIRE_NAME = "@elastic/datemath"
 
-SOURCE_FILES = [
+SOURCE_FILES = glob([
   "src/index.ts",
-]
+])
 
 SRCS = SOURCE_FILES
 
 filegroup(
   name = "srcs",
-  srcs = glob(SOURCE_FILES),
```

Comment:
just a little standard semantic change on the build file. The result will be the same

Reply:
@elastic/kibana-app-services can I have your code owners review on that little change?